jenkins-artifacts

```diff
@@ -62,8 +62,9 @@ function send_jenkins_artifacts ()
   local IS_DIR=$(test -d $SRC && echo "/")
   local DES_DIR=$DES
   if [ "X$IS_DIR" = "X" ] ; then DES_DIR=$(dirname $DES) ; fi
+  if [ $(uname -n | grep "cern.ch" | wc -l) -eq 0 ]; then SSH_TUNNEL="-J cmsbuild@lxplus.cern.ch" ; fi
```

Comment:
this can be moved out side the function e.g. move it on top of the script and update `SSH_OPTS` to include it ( this way you do nothave to change ssh command every where in the script